doc/getting-started.md

```diff
@@ -139,6 +139,122 @@ $ curl -X 'GET' \
 {"_id":"615f30020eb7c3c6616e5ac3","username":"bob","hashed_password":"$2b$12$VtfVij6zz20F/Qr0Ri18O.11.0LJMMXyJxAJAHQbKU0jC96eo2fr.","active":true}
 ```
 
+### Create a Node
```

Comment:
I think this should be put into a separate section about Node object as it's not really something users have to know in order to get started.  A typical starting point is to have the API and pipeline containers running rather than using `curl` with the raw API.

Reply:
Added separate sections. Thanks.